packages/server/test/unit/project_spec.js

```diff
@@ -423,6 +430,34 @@ This option will not have an effect in Some-other-name. Tests that rely on web s
       })
     })
 
+    it('gets app studio for the project id if CYPRESS_ENABLE_CLOUD_STUDIO is set', function () {
+      process.env.CYPRESS_ENABLE_CLOUD_STUDIO = '1'
+
+      return this.project.open()
```

Comment:
can we refactor these tests to use async/await?

Reply:
Done: [`634109a` (#31033)](https://github.com/cypress-io/cypress/pull/31033/commits/634109a70223aa69af77b1fa5415ce3891556128)